src/components/App.tsx

```diff
@@ -26,21 +27,22 @@ type RouteItem = {
 
 // todo improve fallback
 const fallback = <div>loading....</div>;
-const GetStartedPage = React.lazy(() => import('./app-nav-menu/get-started/GetStartedPage'));
+const GetStartedPage = React.lazy(() => import('./app-nav-menu/getting-started/GetStartedPage'));
 const WorkspacesList = React.lazy(() => import('./app-nav-menu/workspaces/WorkspacesList'));
 const Administration = React.lazy(() => import('./app-nav-menu/administration/Administration'));
 const WorkspaceDetails = React.lazy(() => import('./workspace-details/WorkspaceDetails'));
 const IdeIframe = React.lazy(() => import('./ide-iframe/IdeIframe'));
 
 const items: RouteItem[] = [
-  { to: '/', component: GetStartedPage, label: 'Get Started Page', ico: 'fa fa-plus' },
+  { to: '/get-started', component: GetStartedPage, label: 'Get Started Page', ico: 'fa fa-plus' },
```

Comment:
/get-started  ->  getting-started (like './app-nav-menu/getting-started/GetStartedPage')

Reply:
WDYT?

Reply:
I reverter that renaming.